blog/2018/12/20/mojolicious-serving-angular-web-app/index.markdown

```diff
@@ -0,0 +1,379 @@
+---
+title: Day 20: Mojolicious serving Angular Web App
+disable_content_template: 1
+tags:
+  - advent
+  - Mojolicious
+  - Angular
+author: Sachin Dangol
+images:
+  banner:
+    src: '/blog/2018/12/20/mojolicious-serving-angular-web-app/banner.jpg'
```

Comment:
This banner cannot be used as it is not licensed for reuse. Can you please choose something either searching google images with the "licensed for reuse" search tool option or else find something on wikimedia (which are all available).